<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/nicos-gui.qrc">
    <normaloff>:/sunny</normaloff>:/sunny</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="centralLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Application</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionFont"/>
    <addaction name="actionColor"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>&amp;Windows</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBarMain">
   <property name="windowTitle">
    <string>Connection</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBarWindows">
   <property name="windowTitle">
    <string>Windows</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../resources/nicos-gui.qrc">
     <normaloff>:/connect</normaloff>
     <normalon>:/disconnect</normalon>:/connect</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect to server...</string>
   </property>
   <property name="iconText">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Establish or break connection to server</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../../resources/nicos-gui.qrc">
     <normaloff>:/quit</normaloff>:/quit</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>&amp;Font...</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>&amp;Color...</string>
   </property>
  </action>
  <action name="actionUserEditor">
   <property name="icon">
    <iconset resource="../../../resources/nicos-gui.qrc">
     <normaloff>:/editor</normaloff>:/editor</iconset>
   </property>
   <property name="text">
    <string>User &amp;editor</string>
   </property>
   <property name="iconText">
    <string>Editor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionAnalysis">
   <property name="icon">
    <iconset resource="../../../resources/nicos-gui.qrc">
     <normaloff>:/plotter</normaloff>:/plotter</iconset>
   </property>
   <property name="text">
    <string>Data &amp;Analysis</string>
   </property>
   <property name="iconText">
    <string>Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionErrorWindow">
   <property name="icon">
    <iconset resource="../../../resources/nicos-gui.qrc">
     <normaloff>:/errors</normaloff>:/errors</iconset>
   </property>
   <property name="text">
    <string>Errors and warnings</string>
   </property>
   <property name="iconText">
    <string>Errors</string>
   </property>
   <property name="toolTip">
    <string>Show window with all warnings and errors</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionLiveData">
   <property name="icon">
    <iconset resource="../../../resources/nicos-gui.qrc">
     <normaloff>:/live</normaloff>:/live</iconset>
   </property>
   <property name="text">
    <string>Live data</string>
   </property>
   <property name="iconText">
    <string>Live</string>
   </property>
   <property name="toolTip">
    <string>Show live data window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionLogger">
   <property name="icon">
    <iconset resource="../../../resources/nicos-gui.qrc">
     <normaloff>:/find</normaloff>:/find</iconset>
   </property>
   <property name="text">
    <string>History viewer</string>
   </property>
   <property name="iconText">
    <string>History</string>
   </property>
   <property name="toolTip">
    <string>Show data logging and history window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../resources/nicos-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
